int8 sp_enc_left
int8 sp_enc_right
int8 enc_left
int8 enc_right
float64 sp_wheel_left
float64 sp_wheel_right
float64 wheel_left
float64 wheel_right
float64 sp_v
float64 sp_w
float64 v
float64 w